recipes/bob.learn.boosting/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: bob.learn.boosting
+  version: {{ version }}
+
+source:
+  fn: bob.learn.boosting-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.learn.boosting/bob.learn.boosting-{{ version }}.zip
+  md5: d02c5be0c59f55587409a558a98bf04e
+
+build:
+  entry_points:
+    - boosting_example.py = bob.learn.boosting.examples.mnist:main
+  number: 0
+  skip: true  # [not linux]
```

Comment:
I think we made one of its dependencies Linux only, but I'm not remembering right now.